src/main/java/com/google/codeu/servlets/MessageFeedServlet.java

```diff
@@ -0,0 +1,42 @@
+package com.google.codeu.servlets;
+
+import java.io.IOException;
+import java.util.List;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.google.codeu.data.Datastore;
+import com.google.codeu.data.Message;
+import com.google.gson.Gson;
+
+
+/**
+ * Handles fetching all messages for the public feed.
+ */
+
+@WebServlet("/feed")
+public class MessageFeedServlet extends HttpServlet{
```

Comment:
Please keep style consistent with the other files:

`HttpServlet {`
Two spaces for indentation.